ISISScientificComputing/autoreduce
# Add graph of execution time

**Description of changes**


-  Adds a very basic graph as explained in #57.
    - Adds link to this in navbar if you're an admin
- Adds a very basic pie chart to `/stats` that show the status of all of every run in the database.


If this feature proves useful then we'll make it look pretty.

**How to test**

- Click the 'graphs' link in the navbar (displayed for admins only)
- Check the instrument graphs are displaying the correct data, in the correct order


Fixes #57 

---


**To test**
* Log into the devolpement nodes
* `git checkout this-pull-request-branch`
* restart Apache

Once you are happy, merge this pull request into develop, and update the development nodes to the devlop branch
```
> git checkout origin/devlop
> git fetch -p
> git pull origin/develop
```
